enzyme/test/Enzyme/ForwardModeVector/maxnum.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -early-cse -simplifycfg -S | FileCheck %s
+
+%struct.Gradients = type { double, double }
+
+; Function Attrs: nounwind
+declare %struct.Gradients @__enzyme_fwddiff(double (double, double)*, ...)
```

Comment:
For the multi-arg tests, can you add versions where only one argument is active?

Reply:
done